modules/github-runners/README.md

```diff
@@ -57,46 +53,57 @@ components:
               weighted_capacity: null
 ```
 
+## Configuration
+
+### API Token
+
+Prior to deployment, the API Token must exist in SSM.
+
+To generate the token, please follow [these instructions](https://cloudposse.atlassian.net/l/c/N4dH05ud). Once generated, write the API token to the SSM key store at the following location within the same AWS account and region where the GitHub Actions runner pool will reside.
```

Comment:
Not every user of this upstreamed repo will have access to our internal doc. Can we convert these instructions to markdown ?